### ISSUE TITLE
Nothing Showing Up in Drop Down Menu

### ISSUE THREAD

**Issue by wfk-tokunaga** [jugeeya/UltimateTrainingModpack#44]
When I hold the home button to get to the drop down menu for training mode, the menu opens up but nothing appears under the "Training Modpack Menu".

**Comment by jugeeya** [comment]
I'm getting a lot of reports of this, seems to be because of Atmosphere 0.9.2, maybe they dropped support for one of the functions I'm using /: I'll take a look when I have a chance.

**Comment by nharvey27** [comment]
Any update to this I'm using? 1.5 and am having the same issue

**Comment by DIO-ui** [comment]
are the beta and 1.5 versions not compatible with 0.11.0 of atmosphere? I'm getting the same problem where nothing appears under the menu.

**Comment by jugeeya** [comment]
0.11.0 apparently has issues with SaltySD in general, so it may not work with the Training Modpack at all, either until Atmosphere fixes the bug or I finish migration to a different framework.

**Comment by jugeeya** [comment]
@DIO-ui This issue is fixed in the updated 1.5 and beta releases. Please download latest Atmosphere 0.11.1 as well.

**Comment by DIO-ui** [comment]
thank you so much!

**Comment by Areidz** [comment]
For me the game is freezing with the latest SaltySD, the beta release, and Atmosphere 0.11.1 (And the options in the dropdown menu aren't there)

**Comment by Cloud-Galaxyman** [comment]
Seriously, Jugeeya, I'm sorry we only come with problems. Your mod pack has been amazing.
I can't get it to work with Atmosphère  0.12.0 or 11.1.0. 
I'm using Emunand- with Kosmos (15.6-16.1)- and I've tried Switch firmwares 9.2, 10.0.0, 10.0.1, and 10.0.2 with no luck. It crashes on startup in 10.0.2 (and 10.0.1, I think), but 9.2 (and 10.0.0, I think) crash on loading the game (SSBU).
I've tried both the beta and the updated 1.5 release with no luck. I try to clean out the install files each time.
The problem could totally be on my end (like trying to use Kosmos). And in that case, ignore this.
I've attached the crash reports. 

[01588558699_0100000000001005.log](https://github.com/jugeeya/UltimateTrainingModpack/files/4606601/01588558699_0100000000001005.log)
[01588558700_0100000000001005.log](https://github.com/jugeeya/UltimateTrainingModpack/files/4606602/01588558700_0100000000001005.log)
